combo_cases_and_deaths/params.json.template

```diff
@@ -1,13 +1,33 @@
 {
   "common": {
     "log_exceptions": false,
-    "export_dir": "./receiving"
+    "export_dir": "./receiving",
+    "log_filename": "./indicator-combination.log"
```

Comment:
Not sure we want logs saved to a log file for local/dev runs (and etc for other template params). I believe it keeps them from being printed to std out. But also no strong feelings one way or the other.